styles/variables-carbon.less

```diff
@@ -40,6 +40,7 @@
 
 // Panels
 // ---------------------------------------
+@agate-panels-panel-padding: 9px;
```

Comment:
Is there any way to fix this problem without this change?

Reply:
yes, moving the dropdown button position in the sampler. What do you think?

Reply:
Why did this happen? I'm seeing left: 12px style. Shouldn't it be 0?

Reply:
Without adding this padding, the contextual popup is not aligned with the button:
![image](https://user-images.githubusercontent.com/63335068/108373298-6b463480-7208-11eb-8f60-ae995d056b8d.png)
I don't know what is causing this.
If the button is moved in the sampler to the right, the problem dissapears
![image](https://user-images.githubusercontent.com/63335068/108373784-f0c9e480-7208-11eb-981d-1af470ddfbb3.png)

What do you think?

Reply:
I found that ContextualPopupDecorator has its own "KEEPOUT" space, 12px, to avoid the edge of the screen. I think moving dropdown itself a little bit away from the edge will be fine for this case.
Let's remove this variable and make a change to the sample side.

Reply:
alright, I will update the sampler